src/Symfony/Bundle/FrameworkBundle/Controller/PreviewErrorController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Controller;
+
+use Symfony\Component\ErrorRenderer\ErrorRenderer;
+use Symfony\Component\ErrorRenderer\Exception\FlattenException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
```

Comment:
This one could be part of `ErrorRenderer` or `HttpKernel`.

Reply:
merged into ErrorController in dedicated method